package.json

```diff
@@ -389,6 +389,7 @@
     "elliptic": "6.5.4",
     "serialize-to-js": "3.1.1",
     "netmask": "2.0.1",
-    "ssri": "8.0.1"
+    "ssri": "8.0.1",
+    "jose": "1.28.1"
```

Comment:
Let's remove this resolution before merging. It's not needed since this is a safe upgrade.